about.css

```diff
@@ -0,0 +1,106 @@
+main h1 {
+  background-image: none;
+  -webkit-text-fill-color: rgb(236, 82, 66);
+  font-size: calc(1.425rem + 2.1vw);
+  padding-bottom: 1.5rem;
+  font-weight: 700;
+}
+
+.free-courses h2 {
+  font-family: 'Ubuntu', sans-serif !important;
```

Comment:
- [ ] Using `!important` is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets.